starport/cmd/module-create.go

```diff
@@ -86,6 +90,35 @@ func createModuleHandler(cmd *cobra.Command, args []string) error {
 		options = append(options, scaffolder.WithIBCChannelOrdering(ibcOrdering), scaffolder.WithIBC())
 	}
 
+	// Get module dependencies
+	dependencies, err := cmd.Flags().GetStringSlice(flagDep)
+	if err != nil {
+		return err
+	}
+	if len(dependencies) > 0 {
+		var formattedDependencies []modulecreate.Dependency
+
+		// Parse the provided dependencies
+		for _, dependency := range dependencies {
+			splitted := strings.Split(dependency, ".")
+			switch {
+			case len(splitted) == 1:
+				formattedDependencies = append(
```

Comment:
can we avoid duplicating logic between this case and following? we can assing field name to a variable in this switch.

we can assign `len(splitted)` to a var as well.